### PR TITLE
Unify READMEs

### PR DESCRIPTION
**Pull request by samiuelson**
https://stream-io.atlassian.net/browse/CAS-319

### Description

A proposal to keep separate READMEs in `offline` and `client` module directories and redirect to them from the root README file.

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Reviewers added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +39,16 @@ git clone git@github.com:GetStream/stream-chat-android.git
 Open the project in Android Studio and set up your emulator (we're using Pixel 3, API 29 at the moment). Note that the Gradle sync process can take some time when you first open the project. 
 
 ## Docs
+Architecture-wise this SDK consists of the following modules / artifacts:
```
Comment (zsmb13) [inline README.md:42]: Simplify:

```suggestion
This SDK consists of the following modules / artifacts:
```

File: README.md [hunk 2]
```diff
@@ -39,11 +39,16 @@ git clone git@github.com:GetStream/stream-chat-android.git
...
+- [Chat client](https://github.com/GetStream/stream-chat-android/tree/master/stream-chat-android-client)
+- [Offline support and `LiveData` APIs](https://github.com/GetStream/stream-chat-android/tree/master/stream-chat-android-offline)
+- [Chat UI/UX](https://github.com/GetStream/stream-chat-android/tree/master/stream-chat-android)
```
Comment (zsmb13) [inline README.md:45]: Should be relative links within the repo instead

File: README.md [hunk 3]
```diff
@@ -39,11 +39,16 @@ git clone git@github.com:GetStream/stream-chat-android.git
...
 
-This library provides:
+The SDK provides:
 
 - A low-level client for making API calls and receiving chat events
-- `ViewModel` for list of channels and `ViewModel` for channel
+- Offline support and LiveData APIs module
+- Ready to use `ViewModel` for list of channels and `ViewModel` for conversation
```
Comment (zsmb13) [inline README.md:51]:
```suggestion
- Ready to use ViewModels for displaying a list of channels and a conversation 
```

File: README.md [hunk 4]
```diff
@@ -53,10 +58,14 @@ This library provides:
...
 
-### Chat API
+### Chat client
 
 The low-level Chat API docs are available for both [Kotlin](https://getstream.io/chat/docs/kotlin/) and [Java](https://getstream.io/chat/docs/java/).
 
+### Offline support and `LiveData` APIs
+
+The Offline support and `LiveData` APIs docs are available for both [Kotlin](https://getstream.io/chat/docs/kotlin/) and [Java](https://getstream.io/chat/docs/java/).
+
```
Comment (zsmb13) [inline README.md:68]: We could link the specific modules again here

File: stream-chat-android-client/README.md [hunk 5]
```diff
@@ -1,10 +1,7 @@
-![latest-version](https://jitpack.io/v/GetStream/stream-chat-android-client.svg)
-[![Build Status](https://travis-ci.com/GetStream/stream-chat-android-client.svg?branch=master)](https://travis-ci.com/GetStream/stream-chat-android-client)
-[![codecov](https://codecov.io/gh/GetStream/stream-chat-android-client/branch/master/graph/badge.svg)](https://codecov.io/gh/GetStream/stream-chat-android-client)
 
 # Stream Chat Client
 
-`Stream Chat Client` is the official low-level Android SDK for Stream Chat, a service for building chat and messaging applications. This library supports both Kotlin and Java usage. If you can choose we recommend using Kotlin.
+`Stream Chat Client` is the official low-level Android SDK for [Stream Chat](https://github.com/GetStream/stream-chat-android), a service for building chat and messaging applications. This library supports both Kotlin and Java usage. If you can choose we recommend using Kotlin.
```
Comment (zsmb13) [inline stream-chat-android-client/README.md:4]: Be more opinionated 😄 

```suggestion
`Stream Chat Client` is the official low-level Android SDK for [Stream Chat](https://github.com/GetStream/stream-chat-android), a service for building chat and messaging applications. This library supports both Kotlin and Java usage, but we strongly recommend using Kotlin.
```

**Review by zsmb13 — DISMISSED** [review]

**Review by JcMinarro — APPROVED** [review]

**Review by JcMinarro — APPROVED** [review]

**Review by JcMinarro — APPROVED** [review]

**Pull request merged**